### ISSUE TITLE
Cannot easily change game locale / language

### ISSUE THREAD

**Issue by bitcraft** [Tuxemon/Tuxemon#153]
There should be a menu to change the game's language.  I'll outline the steps that I think should be done to complete this issue.  Feel free to comment or implement them.

* Modify locale information to include the name of the locale in target language, ie "English", "Espanol"
* Include a preferred font in the locale info
* Include font glyph sizes in locale info
* Create options menu and link it to start screen and the main menu
* Code to scan locales and dynamically add options
* Code to write configuration file with preferred locale

So, in brief, locale will need some extra options like a preferred font and font size for some languages such as Japanese, Chinese, Thai, etc.  New options will need to be created, and there will need to be ways to save the preferred locale.

**Comment by buddja** [comment]
I think these should be placed in the World State options menu. Then we could change game language/font mid-game.

**Comment by bitcraft** [comment]
Yes, it should be accessible from the World State, as well as the start screen.

**Comment by chagara** [comment]
Helo,

Do you guys need help translating? I can do Spanish.

**Comment by MirceaKitsune** [comment]
Please see also #600 which is related to this. The way translations are stored internally are also a problem, due to there being a single file per language which contains everything (from builtin tags to map-specific dialogue). It's a limitation in terms of creating custom games, as you need to include your own items / monsters / dialogues while staying up to date with other changes.
